lib/player.js

```diff
@@ -4089,7 +4091,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     }
 
     if (this.loadMode_ == shaka.Player.LoadMode.SRC_EQUALS) {
-      if (mimeType == 'text/srt') {
+      if (mimeType != 'text/vtt') {
```

Comment:
While you're here, I'd suggest removing the try block since we shouldn't swallow unknown exceptions and remove the following if since it will never be hit.

Reply:
Done!